assets/questions/sortItinerary.js

```diff
@@ -0,0 +1,137 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Sort Scrambled Itinerary.
+ */
+
+globalData.questions['sortItinerary'] = {  // eslint-disable-line dot-notation
+  title: 'Sort Scrambled Itinerary',
+  starterCode: {
+    python:
+`def sortItinerary(tickets):
+    return ""
+`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+    @classmethod
+    def justConnectTickets(cls, s):
+        return "-".join(s.split(","))
+`
+  },
+  tasks: [{
+    instructions: [
+      [
+        'Whoops! Airline tickets for trips of your boss are messed up! ',
```

Comment:
While I appreciate the narrative, generally the tone for these instructions are instructive, rather than telling a story. Could you rewrite your instructions to be more imperative, like the other questions that are currently in the repo?